sdk/compiler/damlc/tests/BUILD.bazel

```diff
@@ -495,8 +495,6 @@ da_haskell_test(
         "//test-common:upgrades-TemplateChangedKeyType-files",
         "//test-common:upgrades-ValidUpgrade-files",
         "//test-common:upgrades-WarnsWhenAnInterfaceAndATemplateAreDefinedInTheSamePackage-files",
-        "//test-common:upgrades-WarnsWhenAnInterfaceIsDefinedAndThenUsedInAPackageThatUpgradesIt-files",
-        "//test-common:upgrades-WarnsWhenAnInterfaceIsUsedInThePackageThatItsDefinedIn-files",
```

Comment:
Isn't the second test still relevant, since the template is also newly defined? Maybe I misunderstand, I thought the new limit was new interface instances on upgrading templates only

Reply:
It is! I had misread it and thought it was adding a newly introduced interface and to an existing template. So it shouldn't have been failing and there was indeed a bug in my code, which is also a bug in #19817.